device/arista/x86_64-arista_7060x6_64pe/Arista-7060X6-64PE-256x100G/hwsku.json

```diff
@@ -0,0 +1,202 @@
+{
+    "interfaces": {
+        "Ethernet0": {
+            "default_brkout_mode": "8x100G"
+        },
+        "Ethernet8": {
+            "default_brkout_mode": "2x400G"
```

Comment:
does this mean the port names are not continuous when in use? e.g. - Ethernet0, Ethernet1, ...,  Ethernet7, Ethernet16, Ethernet17, ... Ethernet23, ....

Reply:
Correct, here is an example of the output:
```
$ show int sta
  Interface            Lanes    Speed    MTU    FEC         Alias    Vlan    Oper    Admin                           Type    Asym PFC
-----------  ---------------  -------  -----  -----  ------------  ------  ------  -------  -----------------------------  ----------
  Ethernet0               17     100G   9100     rs   Ethernet1/1  routed      up       up  OSFP 8X Pluggable Transceiver         off
  Ethernet1               18     100G   9100     rs   Ethernet1/2  routed      up       up  OSFP 8X Pluggable Transceiver         off
  Ethernet2               19     100G   9100     rs   Ethernet1/3  routed      up       up  OSFP 8X Pluggable Transceiver         off
  Ethernet3               20     100G   9100     rs   Ethernet1/4  routed      up       up  OSFP 8X Pluggable Transceiver         off
  Ethernet4               21     100G   9100     rs   Ethernet1/5  routed      up       up  OSFP 8X Pluggable Transceiver         off
  Ethernet5               22     100G   9100     rs   Ethernet1/6  routed      up       up  OSFP 8X Pluggable Transceiver         off
  Ethernet6               23     100G   9100     rs   Ethernet1/7  routed      up       up  OSFP 8X Pluggable Transceiver         off
  Ethernet7               24     100G   9100     rs   Ethernet1/8  routed      up       up  OSFP 8X Pluggable Transceiver         off
  Ethernet8          1,2,3,4     400G   9100    N/A   Ethernet2/1  routed    down     down                            N/A         off
 Ethernet12          5,6,7,8     400G   9100    N/A   Ethernet2/5  routed    down     down                            N/A         off
 Ethernet16                9     100G   9100     rs   Ethernet3/1  routed      up       up  OSFP 8X Pluggable Transceiver         off
 Ethernet17               10     100G   9100     rs   Ethernet3/2  routed      up       up  OSFP 8X Pluggable Transceiver         off
...
```

Reply:
Down ports removed in latest push to PR; marking resolve.